Requisicao de Ações/servidor.py

```
@@ -0,0 +1,41 @@
+
+# Monitoramento contínuo
+while True:
+    for ticker in acoes:
+        dados = obter_dados_acao(ticker)
+        if dados:
+            mensagem = json.dumps(dados)
+            canal.basic_publish(exchange='', routing_key='stock_updates', body=mensagem)
+            print(f"Publicado: {mensagem}")
+    time.sleep(10) # Intervalo de 10 segundos para evitar excesso de requisições
+
+connection.close()
```

Ruff (0.8.0)

Undefined name `connection`

(F821)